### ISSUE TITLE
Bio.Alphabet has been removed from Biopython

### ISSUE THREAD

**Issue by sgarciah12** [DessimozLab/f1000_PhylogeneticTree#2]
Dear @alpae,

Sorry to bother you again, but I just wanted to report that the script `concat_alignments.py` no longer works in the new Biopython versions, because Bio.Alphabet has been removed and the script makes use of it.

So running it reports the following error:

```
srun python3 concat_alignments.py COGs_for_tree_155/Alignments/Trimmed/*.fasta --format-output fasta > COGs_155_concatenated.fasta
srun: job 1699293 queued and waiting for resources
srun: job 1699293 has been allocated resources
Traceback (most recent call last):
  File "concat_alignments.py", line 8, in <module>
    from Bio.Alphabet import IUPAC, SingleLetterAlphabet
  File "/pasteur/appa/homes/sgarciah/.local/lib/python3.8/site-packages/Bio/Alphabet/__init__.py", line 20, in <module>
    raise ImportError(
ImportError: Bio.Alphabet has been removed from Biopython. In many cases, the alphabet can simply be ignored and removed from scripts. In a few cases, you may need to specify the ``molecule_type`` as an annotation on a SeqRecord for your script to work correctly. Please see https://biopython.org/wiki/Alphabet for more information.
srun: error: maestro-1039: task 0: Exited with exit code 1
```

I understand that adapting the script might take time for you so I will try to develop my own python script for the concatenation of the markers into the supermatrix. But I just wanted to report that the script of the protocol does not run without BioAlphabet.

Thanks!

Samuel

**Comment by alpae** [comment]
Hi @sgarciah12 ,

this problem should already have been resolved since a month with release of version 1.1. Please update the file to the latest version of the git repo, then it should work again.

Best Adrian

**Comment by sgarciah12** [comment]
Hello,

Yes, thanks, I did not realize there was an update. The new one works perfectly fine.

Thank you for your time.

Best

Samuel
